website/pages/docs/platform/k8s/helm/secure.mdx

```diff
@@ -0,0 +1,108 @@
+---
+layout: docs
+page_title: Securing Vault - Kubernetes
+sidebar_title: Securing Vault
```

Comment:
Each of the sidebar titles include "Vault".  This seems redundant because this is the Vault documentation.  I wonder if it makes sense to just make these "Installing", "Unsealing", "Securing", "Upgrading".